project/simpson0114/README.rst

```diff
@@ -0,0 +1,66 @@
+==================================================
+Minimum Spanning Tree Algorithm
+==================================================
+
+Basic Information
+=================
+**Minimum Spanning Tree(MST)** is a significant graph algorithm structure. In a connected, edge-weighted undirected graph, 
```

Comment:
Please also add the github repository in the proposal document.

Reply:
Could you make a comment to point out where you add the github repository link in this document?

Reply:
In the reference part